### PR TITLE
Patternlab/DP-10012  relationship indicators integration MF

### PR DESCRIPTION
**Pull request by clair0917**
## Description
Integration updates for the relationship indicators in Drupal

## Related Issue / Ticket

- [DP-10012](https://jira.mass.gov/browse/DP-10012)

## Screenshots
screen shot 2018-08-15 at 12 10 03 pm
screen shot 2018-08-15 at 12 10 47 pm

### CODE REVIEW

File: assets/scss/02-molecules/_relationship-indicators.scss [hunk 1]
```diff
@@ -69,65 +41,23 @@
...
-            }
+        width: 100%;
+        min-width: 50%;
+        display: flex;
+        flex-direction: column;
+
+        @media ($bp-x-small-min) {
+            flex-flow: row nowrap;
+
+            // &.tags-open {
```
Comment (tnathanjames) [inline assets/scss/02-molecules/_relationship-indicators.scss]: Commented code should likely be removed.

File: assets/scss/00-base/_variables.scss [hunk 2]
```diff
@@ -10,3 +10,4 @@ $layout-sidebar-width: ($column * 3) + ($gutter * 2);
 $layout-sidebar-width-reversed: 370px;
 
 $l-max-content-width: 820px;
+$l-max-content-wrapper: 1240px;
```
Comment (tnathanjames) [inline assets/scss/00-base/_variables.scss]: Line down at the end of the file prevents git from complaining :)

**Review by tnathanjames — APPROVED** [review]
LGTM.

File: assets/scss/02-molecules/_relationship-indicators.scss [hunk 3]
```diff
@@ -201,5 +136,85 @@
...
+            @media ($bp-medium-min) {
+                align-self: baseline;
+                margin-top: 8px;
+            }
+            
+            .tag-state {
+                text-transform: none;
+            }
+            
+            &:after {
```
Comment (Tregonia) [inline assets/scss/02-molecules/_relationship-indicators.scss:180]: Next time you are in this file, we need to change all pseudo elements to use the double `::` and not the single `:`

**Review by Tregonia — APPROVED** [review]
Approved with minor comment

**Pull request merged**